cfme/configure/tasks.py

```diff
@@ -70,39 +158,34 @@ def is_task_finished(destination, task_name, expected_status, clear_tasks_after_
     return True
 
 
-def is_analysis_finished(name, task_type='vm', clear_tasks_after_success=True):
-    """ Check if analysis is finished - if not, reload page"""
+def wait_analysis_finished_multiple_tasks(
+        task_name, task_type, expected_num_of_tasks, delay=5, timeout='5M'):
+    """ Wait until analysis is finished (or timeout exceeded)"""
+    row_completed = []
+    # get view for reload button
+    view = navigate_to(Tasks, 'AllTasks')
+
+    def tasks_finished(output_rows, task_name, task_type, expected_num_of_tasks):
 
-    tabs_data = {
-        'container': {
-            'tab': 'AllTasks',
-            'task': '{}',
-            'state': 'finished'
-        },
-        'vm': {
-            'tab': 'AllTasks',
-            'task': 'Scan from Vm {}',
-            'state': 'finished'
-        },
-        'host': {
-            'tab': 'MyOtherTasks',
-            'task': "SmartState Analysis for '{}'",
-            'state': 'Finished'
-        },
-        'datastore': {
-            'tab': 'MyOtherTasks',
-            'task': 'SmartState Analysis for \[{}\]',
-            'state': "Finished"
-        },
-        'cluster': {
-            'tab': 'MyOtherTasks',
-            'task': 'SmartState Analysis for \[{}\]',
-            'state': "Finished"}
-    }[task_type]
-    return is_task_finished(destination=tabs_data['tab'],
```

Comment:
Looks like this was the only use of this method in the framework so I think you're good to remove it.